client/me/help/help-contact-form/index.jsx

```diff
@@ -93,21 +93,22 @@ export const HelpContactForm = React.createClass( {
 		this.props.valueLink.requestChange( this.state );
 	},
 
-	getSiteSlug() {
-		if ( this.props.selectedSiteSlug ) {
-			return this.props.selectedSiteSlug;
+	getSiteId() {
+		if ( this.props.selectedSiteId ) {
+			return this.props.selectedSiteId;
 		}
 
 		const primarySite = sites.getPrimary();
 		if ( primarySite ) {
-			return primarySite.slug;
+			return primarySite.ID;
 		}
 
 		return null;
 	},
 
 	setSite( siteSlug ) {
-		this.setState( { siteSlug } );
```

Comment:
Ops... apparently I don't know how to add a comment on multiple lines.
I intended to highlight the whole `setSite` function 😞